scheduler/pkg/kafka/config/tls.go

```diff
@@ -63,8 +67,16 @@ func setupSASLSSLAuthentication(config kafka.ConfigMap) error {
 	}
 	config["sasl.username"] = username
 	config["sasl.password"] = ps.GetPassword()
-	// issue is that ca.pem does not work with multiple certificiates defined
+
+	// Set the TLS Certificate
+	cs, err := tls.NewCertificateStore(tls.ValidationOnly(true), tls.ValidationPrefix(EnvKafkaBrokerPrefix))
+	if err != nil {
+		return err
+	}
+	caCert := cs.GetValidationCertificate()
+	// issue is that ca.pem does not work with multiple certificates defined
```

Comment:
:question: Does this comment still apply?

Reply:
did not verify this specificity so I'd keep it for now